### PR TITLE
generate/hcl: Fix an error with the generation

### LINKED ISSUE #29: HCL generation problems
The HCL generation still has some errors and sometimes fails or produces wrong outputs (not right directions or nodes).

We do not have such issue on TFState (we use to but was fixed) but on HCL we have it.

We have to find why is it failing, most probably related on how the HCL->`graph.Graph` before any operation is done to the graph as reading the HCL it's not as easy as TFState(json).

This issues are "hard" to reproduce as they fail once in a while but not always, for example:

```
--- FAIL: TestFromHCL_FlexibleEngine (0.00s)
    --- FAIL: TestFromHCL_FlexibleEngine/SuccessSG (0.00s)
        helper_test.go:66: 
            	Error Trace:	helper_test.go:66
            	            				hcl_test.go:90
            	Error:      	Fail
            	Test:       	TestFromHCL_FlexibleEngine/SuccessSG
            	Messages:   	The Edge with Source "flexibleengine_compute_instance_v2.instance_two" and Target "flexibleengine_compute_instance_v2.instance_one" is present but in the other direction
FAIL
```

### PR DESCRIPTION
**Pull request by xescugc**
Due to only using the 'Variables' interpolation, some Providers where failing on generating HCL
as they also needed some config values that are not present as interpolations on the config/hcl.

With this change we also append to the config all the values to have a more precises config

Closes #29, closes https://github.com/cycloidio/inframap/issues/27

### CODE REVIEW

**Comment by xescugc** [comment]
Fixed the HCL and added the test for `google` HCL

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,11 @@
 - Google graph generation from TFState
   ([Issue #7](https://github.com/cycloidio/inframap/issues/7))
 
+### Fixed
+
+- HCL generation errors
+  ([Issue #29](https://github.com/cycloidio/inframap/issues/29))
+
```
Comment (tormath1) [inline CHANGELOG.md:16]: You can also add this issue: https://github.com/cycloidio/inframap/issues/27

File: generate/hcl.go [hunk 2]
```diff
@@ -216,7 +226,12 @@ func getBodyJSON(b *hclsyntax.Body) map[string]interface{} {
 			// We continue to not add empty information to the config
```
Comment (tormath1) [inline generate/hcl.go:226]: Is this comment still relevant ?

File: generate/testdata/google_hcl.tf [hunk 3]
```diff
@@ -0,0 +1,88 @@
+resource "google_compute_instance" "infraview-tmp" {
+  name         = "infraview-tmp"
```
Comment (tormath1) [inline generate/testdata/google_hcl.tf:2]: mm we should remove `infraview` occurences ? what do you think ?

File: generate/hcl.go (hunk 2 above)
Comment (xescugc) [reply generate/hcl.go:226]: I've moved it to the actual `continue`

**Comment by tormath1** [comment]
ok for rebasing !

**Pull request merged**